### DISCUSSION TITLE
Compatibility with M1(Apple Silicon) Macs

### DISCUSSION THREAD

**Discussion by unggun** [mgtcommerce/mgt-dev#4] [Q&A]
Hello, is there anyone else who has ever tested this on a mac with apple m1 processors? 
I tried but still can't access the elasticsearch and the varnish can't be run.
![image](https://user-images.githubusercontent.com/5086367/112595414-fb495080-8e3c-11eb-83f8-b52c5065f143.png)
![image](https://user-images.githubusercontent.com/5086367/112595649-495e5400-8e3d-11eb-89cc-9ebd91ac1ea6.png)

**Comment by andi-sirclo** [comment]
You got lucky, I got mysql, rabbitmq, and elasticsearch failed to start.

I got mysql started by changing the ownership  of these folder to mysql:mysql
* /var/lib/mysql
* /var/log/mysql
* /var/run/mysql

and changed permission /tmp folder to 777

![image](https://user-images.githubusercontent.com/78332616/112675357-a38d0280-8e99-11eb-97fb-98609f7973f8.png)

**Comment by swieczorek** [comment]
It has not been tested with M1 yet. We will get an M1 soon and will work on it.

**Reply by unggun** [reply]
btw, this elasticsearch image version is working on M1. Maybe this can be useful information.
`docker pull docker.elastic.co/elasticsearch/elasticsearch:7.12.0`
